Sources/Compound/Colors/CompoundColors.swift

```diff
@@ -92,6 +92,27 @@ public struct CompoundColors {
     public let bgSubtleSecondaryLevel0 = Color(UIColor { $0.isLight ? UIColor(compound.colorGray300) : UIColor(compound.colorThemeBg) })
     public let bgCanvasDefaultLevel1 = Color(UIColor { $0.isLight ? UIColor(compound.colorThemeBg) : UIColor(compound.colorGray300) })
     
+    // MARK: - Avatar Colors
+    // Used to determine the background color and the foreground color of an avatar.
+    
+    // Order matches the one from web
+    // https://github.com/vector-im/compound-web/blob/5dda11aa9733462fb8422968181275bc3e9b35e3/src/components/Avatar/Avatar.module.css#L64
+    internal let avatarColors: [AvatarColor] = [
+        .init(background: compound.colorBlue300, foreground: compound.colorBlue1200),
+        .init(background: compound.colorFuchsia300, foreground: compound.colorFuchsia1200),
+        .init(background: compound.colorGreen300, foreground: compound.colorGreen1200),
+        .init(background: compound.colorPink300, foreground: compound.colorPink1200),
+        .init(background: compound.colorOrange300, foreground: compound.colorOrange1200),
+        .init(background: compound.colorCyan300, foreground: compound.colorCyan1200),
+        .init(background: compound.colorPurple300, foreground: compound.colorPurple1200),
+        .init(background: compound.colorLime300, foreground: compound.colorLime1200)
+    ]
+    
+    public func avatarColor(for contentId: String) -> AvatarColor {
```

Comment:
I'll die on this hill damn it!
```suggestion
    public func avatarColor(for contentID: String) -> AvatarColor {
```